docs/docs.jsx

```diff
@@ -2,9 +2,28 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 import {Button} from '../src/Button/Button';
+import {Modal} from '../';
 
 class Demo extends React.Component {
+  constructor() {
+    super()
+    this.state = {modalIsOpen: false};
+  }
+  openModal(){
+    this.setState({modalIsOpen: true});
+  }
+  closeModal() {
+    this.setState({modalIsOpen: false});
+  }
   render() {
+    var modalElement;
+    if(this.state.modalIsOpen){
+      modalElement = <Modal title="Hello Modal" closeModal={this.closeModal.bind(this)}>
```

Comment:
do you need .bind(this)? I haven't done much with es6 classes yet so maybe you do